### DISCUSSION TITLE
2006/11/26/%E3%80%8A%E9%AD%94%E9%AC%BC%E7%BB%8F%E6%B5%8E%E5%AD%A6%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#132] [Announcements]
# 2006/11/26/%E3%80%8A%E9%AD%94%E9%AC%BC%E7%BB%8F%E6%B5%8E%E5%AD%A6%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

这本《魔鬼经济学》是我在地铁口的书摊上买的，当时也是贪个小便宜。回家打开书一看，原来是盗版书，满篇都是错别字，好在还没达到影响理解内容的程度。看来小便宜不能占啊！

https://blog.qizhen.xyz/2006/11/26/%E3%80%8A%E9%AD%94%E9%AC%BC%E7%BB%8F%E6%B5%8E%E5%AD%A6%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Unknown
2007年01月11日 16:06 编辑
读你的文字是一种享受受益非浅！

回复
峰.John
2006年11月28日 23:46 编辑
学习下大哥的思维方式。

回复
竹贤
2006年11月27日 22:03 编辑
“男怕入错行，女怕嫁错郎”，这早就是老皇历了罗。
